src/registrar/utility/csv_export.py

```diff
@@ -1164,6 +1212,39 @@ class DomainDataFederal(DomainExport):
     Inherits from BaseExport -> DomainExport
     """
 
+    # NOTE - this override is temporary.
+    # We are running into a problem where DomainDataFull is
+    # pulling the wrong data.
+    # For example, the portfolio name, rather than the suborganization name.
+    # This can be removed after that gets fixed.
+    # The following fields are changed from DomainExport:
+    # converted_organization_name => organization_name
+    # converted_city => city
+    # converted_state_territory => state_territory
+    # converted_so_name => so_name
```

Comment:
@zandercymatics you had to change senior official too? The portfolios had the wrong SO?  This should ideally pull from the portfolio going forward (as we will _not_ be populating it on future domains), so it's important that if there is an issue with SO we document it and address it.